pkg/controllers/namespace/builder.go

```diff
@@ -6,15 +6,15 @@ import (
 	"github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha1"
 )
 
-func BuildVPCCR(ns string, ncName string, vpcName *string) *v1alpha1.VPC {
-	log.V(2).Info("building vpc", "ns", ns, "nc", ncName, "VPC", vpcName)
-	vpc := &v1alpha1.VPC{}
-	if vpcName == nil {
-		vpc.Name = "vpc-" + uuid.New().String()
-	} else {
-		vpc.Name = *vpcName
-	}
+const (
+	NETWORKINFOPREFIX = "nwf-"
+)
 
-	vpc.Namespace = ns
-	return vpc
+func BuildNetworkInfoCR(ns string) *v1alpha1.NetworkInfo {
+	log.V(2).Info("building networkInfo", "ns", ns)
+	networkInfo := &v1alpha1.NetworkInfo{}
+	networkInfo.Name = NETWORKINFOPREFIX + uuid.New().String()
```

Comment:
I feel we should not use uuid in the name? Use "<Namespace name>-NetworkInfo" as name? @dantingl

Reply:
https://github.com/vmware-tanzu/nsx-operator/blob/ebac866c9a055a104e12e8db8ef03ae4be0cccd5/pkg/controllers/namespace/builder.go#L13
Previous vpc is using uuid.

Reply:
Keep it as the same with namespace as disscussed.